ADJ-Internship/BusinessService/Dtos/PurchaseOrderDtos.cs

```diff
@@ -28,4 +33,190 @@ public void CreateMapping(Profile profile)
             profile.CreateMap<CreateOrUpdatePurchaseOrderRq, PurchaseOrder>().IncludeBase<EntityDtoBase, EntityBase>();
         }
     }
+
+    public class OrderDTO : EntityDtoBase, ICreateMapping
+    {
+        public void CreateMapping(Profile profile)
+        {
+            profile.CreateMap<Order, OrderDTO>().IncludeBase<EntityBase, EntityDtoBase>();
+            profile.CreateMap<OrderDTO, Order>().IncludeBase<EntityDtoBase, EntityBase>();
+        }
+
+        [Required]
+        [Display(Name = "PO Number")]
+        [StringLength(10, ErrorMessage = "Cannot be longer than 10 characters")]
+        [RegularExpression("^[a-zA-Z0-9]+$", ErrorMessage = "Letters and numbers only")]
+        public string PONumber { get; set; }
+
+        //DropList from 2010 to 2020
+        //Default value is Current Date
+        [Display(Name = "Order Date")]
+        [DisplayFormat(DataFormatString = "{0:mm/dd/yyyy}", ApplyFormatInEditMode = true)]
+        [NotInThePast(ErrorMessage = "Cannot be set in the past")]
+        public DateTime OrderDate { get; set; } = DateTime.Now;
+
+        [StringLength(30, ErrorMessage = "Cannot be longer than 30 character")]
+        [RegularExpression("^[a-zA-Z0-9]+$", ErrorMessage = "Letters and numbers only")]
+        public string Buyer { get; set; }
+
+        //Default value = USD
+        public Currency Currency { get; set; }
+        public IEnumerable<SelectListItem> Currencies { get; set; }
+
+        //Droplist from 2010 to 2020
+        public string Season { get; set; }
+        public IEnumerable<SelectListItem> Seasons { get; set; }
+
+        [StringLength(30, ErrorMessage = "Cannot be longer than 30 characters")]
+        [RegularExpression("^[a-zA-Z0-9]+$", ErrorMessage = "Letters and numbers only")]
+        public string Department { get; set; }
+
+        [StringLength(30, ErrorMessage = "Cannot be longer than 30 characters")]
+        [RegularExpression("^[a-zA-Z0-9]+$", ErrorMessage = "Letters and numbers only")]
+        public string Vendor { get; set; }
+
+        [StringLength(30, ErrorMessage = "Cannot be longer than 30 characters")]
+        [RegularExpression("^[a-zA-Z0-9]+$", ErrorMessage = "Letters and numbers only")]
+        public string Company { get; set; }
+
+        //Droplist Vietnam-HongKong
+        [Required]
+        public string Origin { get; set; }
+        public IEnumerable<SelectListItem> Origins { get; set; }
```

Comment:
this is also one of the differences between DTO and MVC model.
SelectListItem has no meaning in our business. 
Putting it here causes the business layer to stick with MVC, then the layering means nothing
Try to think for another solution and let me know in the comment